apps/api/src/app.ts

```diff
@@ -77,6 +78,7 @@ export const app = new Hono()
   .route('/media', mediaRouter)
   .route('/organization', organizationRouter)
   .route('/dash', dashAnalyticsRouter)
+  .route('/template', templateRouter)
```

Comment:
this should be `/exercise` and then in the exerciseRouter there should be a `/template` route